node/include/cocaine/detail/service/node/slave/machine.hpp

```diff
@@ -108,12 +109,39 @@ class machine_t : public std::enable_shared_from_this<machine_t> {
 
         metrics::shared_metric<metrics::gauge<std::string>> state;
         metrics::shared_metric<metrics::gauge<std::uint64_t>> uptime;
+        metrics::shared_metric<metrics::gauge<double>> concurrency;
 
         metrics_t(context_t& context, std::shared_ptr<machine_t> parent);
     };
 
     std::unique_ptr<metrics_t> metrics;
 
+    typedef metrics::usts::ewma_t ewma_type;
+
+    struct metrics_data_t {
```

Comment:
Since we have currently only one special metric it seems like an overkill to wrap it with setters/getters/accessors.

It can be simply replaced with struct with direct access.